packages/pharos/src/components/toggle-button-group/pharos-toggle-button.ts

```diff
@@ -20,7 +20,7 @@ export class PharosToggleButton extends PharosButton {
    * @attr selected
    */
   @property({ type: Boolean, reflect: true })
-  public selected = false;
+  public override selected = false;
```

Comment:
Perhaps this no longer needs to be present, if inheriting it from the superclass now?